spec/controllers/hq/partners_controller_spec.rb

```diff
@@ -1,37 +1,23 @@
 require 'rails_helper'
-class NyuksController < Hq::PartnersController; end
 
 RSpec.describe Hq::PartnersController, type: :controller do
-  describe 'authentication' do
-    controller NyuksController do
-      def foobar_action
-        render nothing: true
-      end
-    end
-
-    it 'checks for admin_user session' do
-      routes.draw { get 'foobar_action', to: 'nyuks#foobar_action' }
-      expect(controller).to be_a_kind_of(Hq::PartnersController)
-      expect_any_instance_of(Hq::PartnersController).to receive :authenticate_admin_user!
-      get :foobar_action
-    end
+  specify 'authentication' do
+    expect(Hq::PartnersController.superclass).to eq HqController
```

Comment:
While this accomplishes the goal, I wonder if it would be better to use shared RSpec examples and `it_behaves_like` - e.g. `it_behaves_like 'HqControllerInherited'`. Not only is it more intent-revealing, but it would also exercise the child controller rather than just testing its superclass.